spec/directions/bake_appendix_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::BakeAppendix do
+  let(:doc) do
+    Kitchen::BookDocument.new(document: Nokogiri::XML(<<~HTML
+      <html>
+      <body>
+        <div data-type="chapter">
+          <div data-type="page">
+            <div data-type="document-title">zzzzzzz</div>
+            <section>
+              <div data-type="title">hello</div>
+              <div>world</div>
```

Comment:
Can you add a nested `<section>` tag to see that its title gets converted to `h2`?